packages/aws-rfdk/lib/lambdas/nodejs/lib/configure-spot-event-plugin/test/spot-event-plugin-client.test.ts

```diff
@@ -238,4 +250,111 @@ describe('SpotEventPluginClient', () => {
     // THEN
     await expect(promise).rejects.toEqual(statusMessage);
   });
+
+  test('successful getGroupPoolCollection for groups', async () => {
+    // WHEN
+    // eslint-disable-next-line dot-notation
+    spotEventPluginClient['deadlineClient'].GetRequest = jest.fn().mockResolvedValue(successfulPoolResponse);
```

Comment:
Similar to my comment as below about `deadlineGroups`, I think it's worth creating a corresponding `successfulGroupResponse` to use here.